gobblin-runtime/src/test/java/gobblin/TestSkipWorkUnitsSource.java

```diff
@@ -69,7 +85,8 @@ public void testSkipWorkUnitPersistence(SourceState state) {
         skipCount++;
       }
     }
-    Assert.assertEquals(skipCount, NUMBER_OF_SKIP_WORKUNITS,
+    // Plus one for MultiWorkUnit1, and plus 2 for MultiWorkUnit2
```

Comment:
Since state store only store taskstates, a total count will suffice. It would take an extra run of Gobblin-Job to separate them out.